docs/data/stashcache/install-origin.md

```diff
@@ -176,7 +176,56 @@ If successful, there should be a file at `/tmp/testfile` with the contents of th
 If unsuccessful, you can pass the `-d` flag to `stashcp` for debug info.
 
 
-## Getting help
+Registering the Origin
+----------------------
+To be part of the OSG StashCache Federation, your _origin_ must be
+[registered with the OSG](https://github.com/opensciencegrid/topology#topology).
+To register your resource:
+
+1. Identify the facility, site, and resource group where your _origin_ is hosted.
```

Comment:
See previous comment about just pointing them to the general instructions and telling them to use "XRootD origin server" for the service type.